lib/client.ts

```diff
@@ -112,8 +112,8 @@ class ServiceClientStrictOptions {
 
     this.defaultRequestOptions = Object.assign({
       protocol: 'https:',
-      pathname: '/',
-      timeout: 2000
```

Comment:
Why removing timeout?

Reply:
general stupidity from early morning, hence the wip label 😄